src/ray/common/ray_syncer/ray_syncer-inl.h

```diff
@@ -79,12 +79,17 @@ class NodeState {
       cluster_view_;
 };
 
-class NodeSyncConnection {
+/// This is the base class for the bidi-streaming call and defined the method
+/// needed. A bidi-stream for ray syncer needs to support pushing message and
+/// disconnect from the remote node.
+/// For the implementation, in the constructor, it needs to connect to the remote
+/// node and it needs to implement the communication between the two nodes.
+class RaySyncerBidiReactor {
```

Comment:
looks much cleaner.

Since RaySyncerBidiReactor is not doing too much, would merge RaySyncerBidiReactor into RaySyncerBidiReactorBase make sense?

Reply:
yes because for the class RaySyncer, it's the same logic.